business-registry-dashboard/app/pages/incorporateNowFromNr.vue

```diff
@@ -77,9 +72,14 @@ onMounted(async () => {
       }
     }
 
+    // Load user's affiliated businesses and check if this name request is already affiliated
+    // This prevents unauthorized access to name requests that belong to other users
```

Comment:
I think NRs can be affiliated to multiple accounts, so this doesn't prevent that.